deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -19,6 +19,15 @@
   notify:
     - Restart Nginx
 
+- name: Add jenkins user to docker, sudo and adm groups
+  user: name="{{ jenkins_name }}"
+        groups="{{ item }}"
+        append=yes
+  with_items:
+    - "docker"
+    - "adm"
+    - "sudo"
```

Comment:
We need to ensure that the Jenkins service is restarted after this change goes into effect.